### PR TITLE
add kkl permits

### PR DESCRIPTION
**Pull request by gruppin**

### CODE REVIEW

File: server/api/lib/trees/regional_tree_permit.js [hunk 1]
```diff
@@ -109,14 +114,14 @@ async function saveNewTreePermits(treePermits, maxPermits) {
 
 	const newTreePermits = treePermits.map(tp => {
 		//if tp is not in the hash map of the existing one - add to the new ones
-		const compact_tp = `${tp.attributes[tpc.REGIONAL_OFFICE]}_${tp.attributes[tpc.PERMIT_NUMBER]}_${formatDate(tp.attributes[tpc.START_DATE], MORNING)}`;
+		const compact_tp = `${tp.attributes[tpc.REGIONAL_OFFICE]}_${tp.attributes[tpc.PERMIT_NUMBER]}_${formatDate(tp.attributes[tpc.START_DATE], MORNING, 'YYYY-MM-DD')}`;
```
Comment (florpor) [inline server/api/lib/trees/regional_tree_permit.js:117]: should add 'YYYY-MM-DD' to line 109 for existing permit key maybe?

File: server/api/model/tree_permit_constants.js [hunk 2]
```diff
@@ -31,4 +31,30 @@ exports.TOTAL_TREES = 'total_trees';
 exports.TREES_PER_PERMIT = 'trees_per_permit';
 exports.SENT = 'sent';
 exports.GEOM = 'geom';
-exports.URL = 'url';
\ No newline at end of file
+exports.URL = 'url';
+
+exports.UNSUPPORTED_PLACES = [
```
Comment (florpor) [inline server/api/model/tree_permit_constants.js:36]: what is unsupported places?

File: server/api/lib/trees/regional_tree_permit.js [hunk 3]
```diff
@@ -136,50 +141,104 @@ async function saveNewTreePermits(treePermits, maxPermits) {
 	}
 }
 
+const getValueBySheetName = (row, tree_permit_field, sheetName) => {
```
Comment (florpor) [inline server/api/lib/trees/regional_tree_permit.js:144]: let's discuss splitting this to separate crawler modules for the next places we will hopefully crawl :)

**Review by florpor — CHANGES_REQUESTED** [review]

**Comment by nonZero** [comment]
Hi!

I have a built a small python parser for these docs:
https://github.com/hasadna/israel_tree_removal_permit_scraper/blob/master/doc_parser.py
I don't know if this is useful - but I'll be happy to help with anything (more formats?) if you need.

You can see somewhat normalized results here:
https://github.com/hasadna/israel_tree_removal_permits_csv
and here:
https://github.com/hasadna/israel_tree_removal_permits_csv

Cheers!
Udi

**Pull request merged**